docs/conf.py

```diff
@@ -156,3 +156,5 @@ def filter(self, record: pylogging.LogRecord) -> bool:
     "show-inheritance": True,
     "exclude-members": "__weakref__,__str__,__repr__",
 }
+# --
+rtd_sphinx_search_file_type = "un-minified"
```

Comment:
@nate-axonius do we need this? this should be only for debug /dev

Reply:
Removed.